pyparsing/diagram/__init__.py

```diff
@@ -29,6 +30,32 @@
 T = TypeVar("T")
 
 
+class EachItem(railroad.Group):
+    """
+    Custom railroad item to compose a:
+    - Group containing a
+      - OneOrMore containing a
+        - Choice of the elements in the Each
+    with the group label indicating that all must be matched
```

Comment:
Didn't you decide [here](https://github.com/pyparsing/pyparsing/issues/226#issuecomment-913092001) that the Group labels are only used for result names?

Reply:
Actually, there was even a qualifier in that note about `NotAny`. Short of defining more extensive constructs, these annotations seemed to be the simplest way to add an indication of special matching behavior. To try to distinguish them from results names, I'm making the annotations all uppercase, and enclosed in `[]`'s. The exception cases are:
- `NotAny` - annotated with `[NOT]`
- `FollowedBy` - annotated with `[LOOKAHEAD]`
- `PrecededBy` - annotated with `[LOOKBEHIND]`

I was hoping that the brackets and uppercase might help distinguish these annotations from results names. Fortunately, it should be rare for any of these constructs to also have a results name.

Reply:
Okay, as long as everything is consistent I'm sure it will be fine.